### DISCUSSION TITLE
Update v0.62.0 from "source" installation to latest version

### DISCUSSION THREAD

**Discussion by ucola** [outline/outline#4736] [Self Hosting] [answered]
Hi all

We have an installation from source with version v0.62.0
Now we want to update it to the latest v0.67.2 version. How can we do this? On the documentation we only see how to install it from source, but nothing about how to update it. Anybody can help us? Which steps we need to do? Would be grateful if anyone can help me.

**Comment by Mante-z** [comment]
I've got exact same question, cannot find how to update if using source method.

**Reply by Mante-z** [reply, marked as answer]
@ucola I actually just managed to do it.
I just downloaded the latest source zip (v0.73.1), built it like the instructions say, and then I just migrated my old .env to the new .env (using the new sample and putting in my .env data from previous).  I then took a full backup of the psql db: `pg_dump -U outlineuser -h localhost -p 5433 outline > outline_backup.sql` Finally I started using `yarn start`, and boom all worked fine.
